contracts/base/IEntityCoreFacet.sol

```diff
@@ -12,13 +13,31 @@ interface IEntityCoreFacet {
    * @param _commmissionsBP The commissions (basis points, 1 = 0.1%) to pay the broker, capital provider and Nayms
    * @param _stakeholders The three stakeholders of the policy - capital provider, insured party, broker
    */
+
+
+  function createPolicyWithTranches(
+    //Policy Variables
+    uint256[] calldata _dates,
+    address _unit,
+    uint256 _premiumIntervalSeconds,
+    uint256[] calldata _commmissionsBP,
+    address[] calldata _stakeholders,
+
+    //Tranche variable array
```

Comment:
Please use proper solidity documentation techniques (above function) to document this, not like this